### DISCUSSION TITLE
Pactum-matchers not working when retry() option is used.

### DISCUSSION THREAD

**Discussion by mihirsasmal** [pactumjs/pactum#249] [Q&A] [answered]
Hi, 

In my test I have used pactum matchers like `includes, regex` . It works fine. However, when I use `retry()` , then the matchers stop working.

Any config or changes need to be done in these cases ?

**Comment by mihirsasmal** [comment]
Is there any way I can make the above scenario work ?

**Comment by ASaiAnudeep** [comment, marked as answer]
Thank you for reporting the issue. Fixed in [v3.3.2](https://github.com/pactumjs/pactum/releases/tag/v3.3.2)
